src/DynamoCoreWpf/Views/FileTrust/FileTrustWarning.xaml.cs

```diff
@@ -47,6 +47,11 @@ public FileTrustWarning(DynamoView dynamoViewWindow)
             SetUpPopup();
 
             HomeWorkspaceModel.WorkspaceClosed += CloseWarningPopup;
+            dynViewModel.NotViewingHomeSpace += DynViewModel_NotViewingHomeSpace; ;
```

Comment:
unsubscribe.. But I think this solution is not general enough since it only handles the `NotViewingHomeSpace` case.. I have mentioned reasoning above.

Reply:
Done